lib/file_validators/utils/content_type_detector.rb

```diff
@@ -48,7 +48,7 @@ def empty_file?
       alias :empty? :empty_file?
 
       def blank_name?
-        @filepath.nil? || @filepath.try(:empty?)
+        @filepath.nil? || @filepath.empty?
```

Comment:
If you've got active support loaded, which I assume you will have you can just do,
`@filepath.blank?`